client/src/components/RootLayout.tsx

```diff
@@ -0,0 +1,17 @@
+import { Outlet } from 'react-router-dom';
+import Footer from './Footer';
+import Navbar from './Navbar';
+
+const RootLayout = () => {
+  return (
+    <div className="flex flex-col h-full justify-between">
+      <Navbar />
+      <main className="w-screen h-full flex flex-col justify-center items-center bg-secondary">
```

Comment:
You can remove the `h-full` here or on line 7 to fix the issue with the card on the `Register` page

Reply:
Thank you. It fixed the contents being cutoff. Although, when screen is tall enough, the background did not fill up the space between navbar and footer. I tried various different height settings, and at the end settled on using a vertical breakpoint to toggle `h-full`. Not sure if this is the ideal approach.